src/main/java/model/Ticket.java

```diff
@@ -49,7 +49,7 @@ public Ticket(ConcertHall concertHall, int eventCode, LocalDateTime eventTime, b
 
         this.concertHall = concertHall;
         this.eventCode = eventCode;
-        setEventTime(eventTime);
+        //setEventTime(eventTime);
```

Comment:
remove comments when code is ready for review

Reply:
Fixed